modules/nextflow/src/test/groovy/nextflow/config/parser/v2/ConfigParserV2Test.groovy

```diff
@@ -202,31 +203,31 @@ class ConfigParserV2Test extends Specification {
 
         main. text = '''
             profiles {
-                includeConfig 'dir1/config'
-                includeConfig 'dir2/config'
-                includeConfig 'dir3/config'
+                proc1 {
+                    includeConfig 'dir1/config'
```

Comment:
What about `includeConfig` outside `profiles` is that allowed?

Reply:
yes of course. that is covered by other unit tests

Reply:
Okey